server/tools/peertube-import-videos.ts

```diff
@@ -184,6 +185,8 @@ async function processVideo (parameters: {
   } catch (err) {
     log.error(err.message)
   }
+
+  return false;
```

Comment:
extra `;`

Reply:
Well they were stupid mistakes.
Would it be acceptable to force push the change?
Or do a fresh commit, assuming perhaps that you would anyway squash merge later?

Reply:
Both are usually acceptable.